infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -3,6 +3,10 @@ x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
   dpl-cms-release: "2024.18.0"
+x-early-movers: &early-movers-release-image-source
+  releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
+  releaseImageName: dpl-cms-source
+  dpl-cms-release: "2024.18.2"
```

Comment:
:+1: I like this grouping. 

Is possible to inherit repo and image name from defaults?

Reply:
Maybe, but untested. Will continue to improve groupings/inheritance as we find the right groups over time